Src/Witsml/Data/WitsmlWellboreGeometry.cs

```diff
@@ -0,0 +1,44 @@
+using System.Xml;
+using System.Xml.Serialization;
+using Witsml.Data.Measures;
+
+//Hvor kommer betydning av minOccurs inn?
+
+namespace Witsml.Data.WellboreGeometry
+{
+    public class WitsmlWellboreGeometry
```

Comment:
I think it would be better to call this WitsmlWbGeometry to follow the naming from WITSML. Also a WITSMLWbGeometrys class is missing, akin to WitsmlTubulars.

Reply:
Would it be WITSMLWbGeometrys or WITSMLWbGeometries?